integration/test_collection.py

```diff
@@ -1635,3 +1635,26 @@ def test_collection_shards(collection_factory: CollectionFactory) -> None:
     assert shards[0].collection == collection.name
     assert shards[0].name is not None
     assert shards[0].object_count == 0
+
+
+def test_return_phone_number_property(collection_factory: CollectionFactory) -> None:
+    collection = collection_factory(
+        properties=[
+            Property(name="phone", data_type=DataType.PHONE_NUMBER),
+        ]
+    )
+    if collection._connection._weaviate_version < _ServerVersion(
+        1, 23, 2
+    ):  # TODO: change to 1.23.3 when it is released
+        pytest.skip("Phone number properties are only supported on Weaviate >= 1.23.3")
+    uuid = collection.data.insert(
+        {"phone": PhoneNumber(default_country="GB", number="02070354848")}
```

Comment:
Is this the best/only way to insert phone numbers? Personally I would use the internation format like 
+COUNTRY_CODE NATIONAL_NUMBER

Reply:
>If we accepted, e.g. +44 0161 234 5000, we would need to parse this and convert +44 into GB

This is wrong. We can send anything in `number` and Weaviate will parse it appropriately so the `GB` here is optional. I will add this use as part of the test to show it